source/overview/security.rst

```diff
@@ -79,40 +138,45 @@ Security updates address newly discovered attacks reported to Mattermost, Inc. b
 For more information, please see:
 
 - `Mattermost Security Updates Disclosures <http://about.mattermost.com/security-updates/>`_
-   - A summary of security updates made based on past and on-going security analysis and penetration testing.
+A summary of security updates made based on past and on-going security analysis and penetration testing.
```

Comment:
0/5 - should we combine into one line instead of having the description below? 
<img width="849" alt="screenshot 2017-11-15 19 27 02" src="https://user-images.githubusercontent.com/9354841/32867419-022c3a44-ca3b-11e7-932c-17f01889ceb8.png">

Reply:
yes...done....